pyproject.toml

```
@@ -0,0 +1,9 @@
+[build-system]
+requires = [
+    # pin setuptools on pypy to workaround this bug: https://github.com/pypa/distutils/issues/283
+    "setuptools < 72.2; platform_python_implementation == 'PyPy'",
+    "setuptools ; platform_python_implementation != 'PyPy'",
+    "wheel",
+    "Cython>=3.0"
```

```suggestion
    "Cython==3.0.12"
```

It breaks on cython==3.1, because `long` is no longer supported like in https://github.com/cython/cython/issues/6490

---

Thanks for this comment, I’ve updated the PR to keep using Cython <3.1.